SPCTR-3174: Fix: How To - minutes value is not skippable in duration. (#3428)
brainstormforce/ultimate-addons-for-gutenberg: README.md

### x.x.x ###
* Fix: Modal - Resolved an issue in which the Modal would not open in Astra's quick-view popup for WooCommerce products.
* Fix: Separator - Resolved an edge-case issue in which the responsive alignment would not work as intended.

> * Fix: HowTo - Resolved an issue were the minimum minutes not able to skipped.

Reviewer: ```suggestion
* Fix: How To - Resolved an issue in which the minimum minutes setting would be unskippable.
```

Author: Done.
